test/suite-config/fedprops/LocalSettings.php

```diff
@@ -7,4 +7,24 @@
 $wgDebugDumpSql  = true;
 $wgShowDBErrorBacktrace = true;
 
-$wgWBRepoSettings['federatedPropertiesEnabled'] = true;
\ No newline at end of file
+$wgWBRepoSettings['federatedPropertiesEnabled'] = true;
+$wgWBRepoSettings['federatedPropertiesSourceScriptUrl'] = 'https://www.wikidata.org/w/';
```

Comment:
Do we want to leave a comment here that this can be removed once we drop support for 1.35?

Reply:
yeah sounds good.